provider/pkg/schema/gen.go

```diff
@@ -851,6 +865,16 @@ func (ctx *context) gatherResourceType() error {
 		if err != nil {
 			return err
 		}
+		if hasTags && prop == tagsProp {
```

Comment:
This added block seems to be identical to the one above? Can we extract it?

Reply:
Done 👍